### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by ashokdey**
Changes:

- Fix in the naming of variables
- Better comments
- returning ```null``` instead of ```false```

### CODE REVIEW

File: src/_Problems_/find-2nd-max/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ * You may find it easy but it's tricky for few
+ * Input - [9, 2, 3, 6]
+ * Output - 6
+ */
+
+function findSecondMax(arr) {
+  let max = arr[0];
+  let max2 = 0;
```
Comment (TheSTL) [inline src/_Problems_/find-2nd-max/index.js]: Initialize max2 = -1. If there are no Second Max then function should return -1.
Input : [1,1,1]
Output : -1

File: src/_Problems_/find-2-nums-adding-to-n/index.js [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+    for (let j = i + 1; j < arr.length; j += 1) {
+      if (arr[i] + arr[j] === number) {
+        pair.push(arr[i], arr[j]);
         break;
       }
     }
   }
 
-  return nums.length ? nums : false;
+  return pair.length ? pair : null;
```
Comment (TheSTL) [inline src/_Problems_/find-2-nums-adding-to-n/index.js]: return false if no pair

File: src/_Problems_/find-2nd-max/index.js (hunk 1 above)
Comment (ashokdey) [reply src/_Problems_/find-2nd-max/index.js]: I think initializing and returning ```null``` makes more sense in case of **JavaScript**

Comment (ashokdey) [reply src/_Problems_/find-2nd-max/index.js]: Oh yes, I find something better

```javascript 
let max2 = Number.MIN_SAFE_INTEGER;
```

**Pull request merged**